plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -193,6 +193,14 @@ public void testInsertDuplicateRows()
         }
     }
 
+    @Test
+    @Override
+    public void testInsertNegativeDate()
+    {
+        // TODO (https://github.com/trinodb/trino/issues/10208) Fix negative date handling. Expected '-2016-12-07', but got '-2016-11-21'
+        assertThatThrownBy(super::testInsertNegativeDate);
```

Comment:
assertThatThrownBy should be followed by an assertion over the exception.

sadly, the method is annotated with `@CanIgnoreReturnValue` (because it at least checks that an exception was throw), but it really should be `@CheckReturnValue`